lib/components/map/itinerary-summary-overlay.tsx

```diff
@@ -112,7 +126,8 @@ function getUniquePoint(
 
     const selfDistance = distance(point, centerOfLine)
     // maximize distance from all other points while minimizing distance to center of our own line
-    const averageDistance = totalDistance / otherMidpoints.length - selfDistance
+    const averageDistance =
+      1.2 * (totalDistance / otherMidpoints.length) - selfDistance
```

Comment:
What's this 1.2 do?

Reply:
This prioritizes the labels not overlapping slightly above them being in the center of the line. I'm not super happy with this value, so we'll probably tweak it more in the future